### PR TITLE
Redhat support

### PR DESCRIPTION
**Pull request by andrewnoruk**
I changed to use double quotes so we could use string substitution, added a require for version.rb and fixed the expected bindfs directory path by adding the version number.

### CODE REVIEW

**Comment by ricoli** [comment]
Thank you, my ruby skills ain't great...

File: lib/vagrant-bindfs/cap/redhat/bindfs_install.rb [hunk 1]
```diff
@@ -6,14 +6,17 @@ module BindfsInstall
...
-                comm.sudo('[ -d ./bindfs ] && cd bindfs && ./configure && make && make install')
-                comm.sudo('ln -s /usr/local/bin/bindfs /usr/bin')
+
+              require "vagrant-bindfs/version"
+
+              if comm.test("grep \"CentOS release 6\" /etc/centos-release")
+                comm.sudo("yum -y install fuse fuse-devel")
+                comm.sudo("wget http://bindfs.org/downloads/bindfs-#{VagrantPlugins::Bindfs::SOURCE_VERSION}.tar.gz -O bindfs.tar.gz")
+                comm.sudo("tar --overwrite -zxvf bindfs.tar.gz")
+                comm.sudo("[ -d ./bindfs-#{VagrantPlugins::Bindfs::SOURCE_VERSION} ] && cd bindfs-#{VagrantPlugins::Bindfs::SOURCE_VERSION} && ./configure && make && make install")
```
Comment (ricoli) [inline lib/vagrant-bindfs/cap/redhat/bindfs_install.rb:16]: should be `comm.sudo("[ -d ./bindfs ] && cd bindfs && ./configure && make && make install")`, as I'm outputting the file as bindfs.tar.gz, perhaps you had an old file in the same dir?

Comment (andrewnoruk) [inline lib/vagrant-bindfs/cap/redhat/bindfs_install.rb:16]: Even though you were naming the TAR file bindfs.tar.gz it doesn't change the name of the directory that's archived inside it. In this case, the TAR file included the bindfs-[SOURCE_VERSION] directory.

Comment (ricoli) [inline lib/vagrant-bindfs/cap/redhat/bindfs_install.rb:16]: ah I see, thank you :)

**Pull request merged**